ios/engine/KMEI/KeymanEngine/Classes/kmp.json/KMPKeyboard.swift

```diff
@@ -40,8 +39,9 @@ public class KMPKeyboard: Codable {
     font = try values.decodeIfPresent(String.self, forKey: .font)
     isRTL = try values.decodeIfPresent(Bool.self, forKey: .isRTL) ?? false
     languages = try values.decode([KMPLanguage].self, forKey: .languages)
```

Comment:
Does this lower-case the language ID's?
So we can match `str-Latn` == `str-latn`

Reply:
It does not do so yet, no.  :+1:

Reply:
Latest commit fixes this.